src/encoded/static/components/glossary.js

```diff
@@ -33,6 +33,14 @@ GlossaryTerm.propTypes = {
     glossaryEntry: PropTypes.object.isRequired,
 };
 
+// Optional links appended to section headers
+const sectionHeaderLinks = {
+    'Target categories': {
+        link: 'https://www.encodeproject.org/target-categorization/',
```

Comment:
Is this supposed to be an internal link or external link? With this full URL you’ll get the external-link icon with the arrow after the link, and I wasn’t sure that’s the intention or not.

Reply:
Well, it is an internal link so perhaps we should not be using that icon. We're using this icon for all the links on the glossary page, which are all internal links, so maybe they should all be changed. Looking around on Encode I think we're very inconsistent about our use of the external link icon, sometimes we use it, sometimes we don't. Is there another icon we use for internal links?

Reply:
Internal links get no icon, like the blue text in the search results. Anything in the code with a path and no protocol display no icon (e.g. '/target-categorization/'), but any with a full URL including protocol get the icon automatically, which is why this one displays that icon.

Reply:
Hmm, I see one inconsistency so far: the SCREEN link on the home page should have that icon because the link has a full URL, but the .site-search__screen a:after style deletes it.

Reply:
There are a lot of inconsistencies on the help pages.

Reply:
Oh really? People who edit the pages probably sometimes use paths and sometimes full URLs. Should probably tell everyone to make sure to use paths for on-site targets.

Reply:
I’ll write a WRAN ticket for that, and bring it up in their channel.